### PR TITLE
Fix The Html for better accessibility

### PR DESCRIPTION
**Pull request by DavidKohav**
#336

### CODE REVIEW

**Comment by DavidKohav** [comment]
noerrors

**Comment by DavidKohav** [comment]
hgizra

**Comment by DavidKohav** [comment]
No errors anymore in the homepage.
Please code review.

**Comment by DavidKohav** [comment]
Done

**Comment by DavidKohav** [comment]
Please check

**Comment by amitaibu** [comment]
Probably needs changes after the new homepage design

**Comment by DavidKohav** [comment]
outline
outline

**Comment by DavidKohav** [comment]
accessibility app with 0 errors
b

**Comment by DavidKohav** [comment]
The full page.
a

**Comment by DavidKohav** [comment]
@amitaibu - Done

**Comment by DavidKohav** [comment]
@amitaibu - We need to deal with the Contrast Errors .

File: app/_includes/about.html [hunk 1]
```diff
@@ -24,6 +24,7 @@ <h2 class="title">About us</h2>
         <div class="page-scroll">
           <div class="scroll-wrapper">
             <a href="#gizra-way" class="smooth-scroll-to">
+              <span class="sr-only">look on our agenda</span>
```
Comment (amitaibu) [inline app/_includes/about.html]: what do you mean by `look on our agenda`?

**Comment by amitaibu** [comment]
@IshaDakota can you please review

**Comment by IshaDakota** [comment]
Looks good. I renamed the screen reader labels for scroll items. 

@DavidKohav Do you also want to take a look at #443 ? I'm afraid if I do it, i'll break the good work you did here.

**Comment by amitaibu** [comment]
Adam, If ok please deploy

**Comment by DavidKohav** [comment]
@IshaDakota - I will check #443

**Comment by IshaDakota** [comment]
I'll deploy after #443

**Comment by DavidKohav** [comment]
https://gfycat.com/PersonalPerkyAfricanwildcat

**Comment by DavidKohav** [comment]
Done

**Comment by IshaDakota** [comment]
Lovely. Merged and deployed.

**Pull request merged**